public/app/model-loader.js

```
@@ -1,287 +1,133 @@
+  { id: 'bsrnn',      filename: 'bsrnn_quantized.onnx',     priority: 'lazy'  },
+];
 
-  const response = await fetch(url, { cache: 'no-store' });
-  if (!response.ok) {
-    const err = `HTTP ${response.status} fetching ${filename}`;
-    _broadcast({ type: 'error', id, filename, error: err });
-    throw new Error(err);
-  }
+function broadcast(type, model, progress) {
+  try {
+    const bc = new BroadcastChannel(BROADCAST_CH);
```

ESLint

[error] 'BroadcastChannel' is not defined.

(no-undef)